### PR TITLE
Translation to Hebrew

### PR DESCRIPTION
**Pull request by yarons**
Added a Hebrew translation, consider using a localization platform such as Weblate.

### CODE REVIEW

**Review by p0deje — CHANGES_REQUESTED** [review]
This looks great, but there are also Settings views that need translation, can you add those as well https://github.com/p0deje/Maccy/tree/master/Maccy/Settings?

I'm not sure how to use Weblate, but I'll take a look - thank you.

**Comment by yarons** [comment]
Updated, I missed this part of the translation requirements, thanks :)

**Comment by p0deje** [comment]
@yarons Thank you, I completely forgot about one more file, could you add it too? https://github.com/p0deje/Maccy/tree/master/Maccy/Menu/Preview

**Comment by yarons** [comment]
Yeah, that's exactly what localization platforms are good for :)

Thank you for this amazing app!

**Review by p0deje — COMMENTED** [review]
Thank you once again, I've noticed one string could have been missed, can you please take a look?

File: Maccy/Settings/he.lproj/IgnoreSettingsViewController.strings [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+"Rfe-XX-kt3.title" = "תא תצוגת טבלה";
+
+/* Class = "NSButtonCell"; title = "Ignore all applications except listed"; ObjectID = "Zb2-M8-Pky"; */
+"Zb2-M8-Pky.title" = "התעלמות מכל היישומים חוץ מאלו הרשומים";
+
+/* Class = "NSTabViewItem"; label = "Pasteboard types"; ObjectID = "b7c-gy-SzX"; */
+"b7c-gy-SzX.label" = "סוגי לוחות הדבקה";
+
+/* Class = "NSTextFieldCell"; title = "It's possible to ignore copies coming from certain applications.\nPlease note that the way it works is not bullet-proof, so it's better to use pasteboard types when possible."; ObjectID = "cCb-x7-Qin"; */
+"cCb-x7-Qin.title" = "Es ist möglich, kopierte Elemente von bestimmten Anwendungen zu ignorieren.\nBitte beachte, dass diese Methode nicht hundertprozentig zuverlässig funktioniert. Daher ist es besser, wenn möglich Pasteboard-Typen zu verwenden.";
```
[the remark this answers is not in the document]
Comment (yarons) [reply Maccy/Settings/he.lproj/IgnoreSettingsViewController.strings]: Done, my bad.

**Comment by p0deje** [comment]
Thank you once again!

**Comment by yarons** [comment]
Thank you again for this awesome project!

**Pull request merged**